### DISCUSSION TITLE
I wish he could support setting or resizing images, embedding iframe and video

### DISCUSSION THREAD

**Discussion by ac009527** [avitorio/outstatic#79] [Ideas]
I like outtatic very much
I wish he could support setting or resizing images, embedding iframe and video

**Comment by thecodedrift** [comment]
Unfortunately, we're kind of limited to what's possible in Markdown & text (for now). However, the `markdownToHtml` step can be replaced with tools such as MDX, remark, or even Markdoc. That would give you greater control of the resulting HTML and React Components. For example, you can see this in Markdoc's `youtube` component @ https://markdoc.dev/docs/overview

```md
{% youtube
  src="https://www.youtube-nocookie.com/embed/MAWK_VmjU1Y?controls=0"
  title="Introducing Markdoc"
  width="50%" /%}
```

**Reply by ac009527** [reply]
Thank you for your reply, this is enough for me at the moment, I will continue to pay attention to the update of outstatic afterwards.
